software/gui/gui.sh

```diff
@@ -131,15 +134,15 @@ fi
 #########
 # BUILD #
 #########
-
 if [ "$1" == "--build" ]; then
+  cppcheck -ithird_party -ibuild .
```

Comment:
Should this be happening before everything else?
Especially before the root privileges check?
Should this also depend on the `--no-checks` option at all?

Reply:
You're right, moving the `--no-checks` after the root check and then the cppcheck after that